content/docs/user-guide/privacy.md

```diff
@@ -34,8 +34,8 @@ which requires that you authenticate yourself as a specific Google user and
 authorize these actions.
 
 DVC can help you get a token by guiding you through the OAuth flow in the
-browser. There you must consent to allow the _DVC_ Application to operate on
-your behalf. The OAuth consent screen will describe the scope of what is being
+browser. From there you can allow the _DVC_ Application to operate on your
```

Comment:
let's not change this legal language

Reply:
OK, changes undone. Should I avoid touching this file completely going fwd?